src/components/mdxComponents/CtaArticleComponent.js

```diff
@@ -44,35 +46,37 @@ const StyledButton = styled(Button)`
   `}
 `
 
-const CtaArticleComponent = ({ heading, buttonText, onClick }) => {
+const headings = [
+  "Let’s build disruptive JavaScript products together",
+  'Build your modern Web App with top React & Node.js Engineers',
+  ]
+const buttonTexts = ['Join our Team!', "Let's talk!"];
+
+const CtaArticleComponent = () => {
   const { isInPoland } = useContext(CountryContext);
+  const [, showContactModal, hideContactModal] = useModal(ContactModal, { onSubmitContactForm });
```

Comment:
Each time when we use ContactModal, we are putting the same arguments like onSubmitContactForm which is basically the same as hideContactModal function. hideContactModal is returned from useModal function so we have an `'onSubmitContactForm' was used before it was defined.` error. I'm thinking if we can call `hide` function inside `ContactModal` and leave onSubmitContractForm for more case-specific code that we will add in the future.